test/exceptions_test.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group('description', () {
```

Comment:
What's this blank file for?

Reply:
This means waiting to be added.

Reply:
Then this file shouldn't be included in this PR.

Reply:
Got it.